### DISCUSSION TITLE
ERROR

### DISCUSSION THREAD

**Discussion by myles800** [freekmurze/laravel-analytics#374] [Q&A] [answered]
Hi,
I implemented this library in mij laravel project, but i get this error. 
{ "error": { "code": 403, "message": "User does not have sufficient permissions for this profile.", "errors": [ { "message": "User does not have sufficient permissions for this profile.", "domain": "global", "reason": "insufficientPermissions" } ] } }

Analitics does not give a view id anymore so, I tried my property id and my account id, but non work.
Can you help me?

**Comment by fahadali1** [comment, marked as answer]
You can find it by clicking on Show advanced options (when you create a new property):

![Capture](https://user-images.githubusercontent.com/40367179/101612177-2d6ca900-3a2c-11eb-871a-560f186f052d.PNG)

**Comment by DevHoracioRodriguez** [comment]
> { "error": { "code": 403, "message": "User does not have sufficient permissions for this profile.", "errors": [ { "message": "User does not have sufficient permissions for this profile.", "domain": "global", "reason": "insufficientPermissions" } ] } }


Hello

How did you solved the error? I did tried the property with "Create a Universal Analytics property" but when I go back to the website the error still there. On .env when change debug to false, will show and allow me to login, then gives me 500 - Internal Server Error.

I'm using https://github.com/ozdemirburak/laravel-8-simple-cms

Any ideas on what to try? 

Thank you

**Reply by salmanaugmentler** [reply]
From the `service_account_credentials.json` file:

1. Copy your `client_email`
2. Go to the desired View that you have connected to your LaravelAnalytics
3. Go to Admin >> User Account Management
4. Add (New User)
5. Paste the `client_email`
6. Select "Analyst" as role
7. Save

That's it. You are good to go.

**Reply by salmanmichsan** [reply]
After I follow your instructions, I still get the error, any idea?

**Reply by tnylea** [reply]
@salmanaugmentler This worked for me. Thanks! I guess I was like all the others and didn't finish reading the instructions in the readme 🤦‍♂️

Anyway! Appreciate it.

**Comment by sanjaydeveloper15** [comment]
Hi 👋

I am getting login required error


I am using json file and view ID both
As per documentation there is no option given where we need to put oauth access token


So do you have any idea regarding it ?

**Comment by Null78** [comment]
i'm getting this with property_id and GA4?, i upgraded to v5 recently
